website/docs/reference/glossary.md

```diff
@@ -0,0 +1,159 @@
+---
+sidebar_position: 2
+---
+
+# Glossary
+
+## List of FerretDB terminologies
+
+*This section contains a list of common terminologies related to FerretDB*.
+
+### A
+
+#### aggregation
+
+A group of operations used to reduce or summarize large data sets.
+See [list of supported aggregation operations and commands here](./supported_commands.md#aggregation-pipelines).
+
+#### aggregation pipeline
+
+A set of operators that lets you perform complex operations that aggregate and summarize values.
+See [list of supported aggregation pipeline operators](./supported_commands.md#aggregation-pipeline-operators) here.
+
+---
+
+### B
+
+#### Beacon
+
+The telemetry service of FerretDB.
```

Comment:
Should we link to the beacon.ferretdb.io or telemetry page?

Reply:
Probably the telemetry page for now. Once the beacon page is ready, we can update the link.

Reply:
The telemetry page is ready :)